web/src/views/Dashboard.tsx

```diff
@@ -16,6 +16,7 @@
 
 import { useKeycloak } from '@react-keycloak/web';
 import React, { useEffect, useState } from 'react';
+import { Bool } from 'reselect/es/types';
```

Comment:
You are not using this Bool why add here?

Reply:
Woops. Removed that.